### DISCUSSION TITLE
Please help, I need to delete the post with the photo.

### DISCUSSION THREAD

**Discussion by Kkleytt** [aiogram/aiogram#1274] [Q&A]
I send a message with media on behalf of the bot, later this message needs to be deleted, also on behalf of the bot, but it doesn’t work, the message was sent by the bot.send_photo command

**Comment by JrooTJunior** [comment]
What does mean "doesn't work"?
Did you get any error?

**Reply by Kkleytt** [reply]
Error: aiogram.utils.exceptions.BadRequest: There is no text in the message to edit
And it doesn’t matter if I try to delete the message or edit it, the error comes out the same, I’m a beginner programmer, so I looked for an answer on the Internet, at ChtaGpt, but all attempts are in vain
